### ISSUE TITLE
Scarab isn't working

### ISSUE THREAD

**Issue by Snarich** [fifty-six/Scarab#116]
### Scarab Version

v1.19.0.0

### Hollow Knight Version

v1.5.78.11833

### Steps to Reproduce

Don't actually know.

### Expected Behaviour

Scarab was expected to work as it should.

### Actual Behaviour

Scarab launched, but gave an error after choosing the path to the game
![image](https://user-images.githubusercontent.com/118846886/203394081-181d727d-6daa-4951-a896-6926963521ec.png)

### Logs

None

**Comment by Awesome-Danger** [comment]
This is also Happening to me too. I have the latest version of Hollow Knight on Steam, I have the Lastest version of Scarab. and I've been modding games for several years now, so I know it isn't a lack of experience with modding in general.

**Comment by Awesome-Danger** [comment]
I fixed it. I downloaded the scarab reset -which the reset is titled on Desktop: "HKModInstaller_Reset" and double clicked it to execute it. When you execute it it will show a window that says to press any button to continue. press any button and it will disappear. it might not look look like it did anything, but it did. I then verified the files on steam (which I did many times already), did NOT put a mods folder in the hollow knight files, then I either ran the game with no mods first and *then* ran scarab, or just ran scarab. Scarab asked me if the location it detected was correct. I said yes, and downloaded one of the first mods I saw and it worked. Hope this helps!

Reset installer link [here.]([https://cdn.discordapp.com/attachment...](https://www.youtube.com/redirect?event=video_description&redir_token=QUFFLUhqbWdMSHZybl9MVHBkaXV1ckpwNzdpa0g0MkpLd3xBQ3Jtc0trWWE2NGktX3dPWUI4UEVpTTR3Xzk3SDJ5enpGWUJVNVFLS3hWM0szdU5TbmZLVG9vWklhZFFSM1FhdEF6Tmh1alR6X1I1d3VtSjF0TEpSVlAxV2l3RWNwTTN1SGw0ZVZxQ0p2MC0zMTlFLVFBanpaWQ&q=https%3A%2F%2Fcdn.discordapp.com%2Fattachments%2F879138066961424405%2F890824593848299550%2FHKModInstaller_Reset.bat&v=z35cFvU0McQ))
